projects/storefrontlib/src/cms-components/myaccount/order/cancellations-returns/return-order/return-order.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+import { tap, filter } from 'rxjs/operators';
+
+import {
+  Order,
+  OrderEntry,
+  CancellationReturnRequestEntryInput,
+  RoutingService,
+} from '@spartacus/core';
+
+import { OrderDetailsService } from '../../order-details/order-details.service';
+
+@Component({
+  selector: 'cx-return-order',
+  templateUrl: './return-order.component.html',
+})
+export class ReturnOrderComponent implements OnInit {
+  constructor(
+    protected orderDetailsService: OrderDetailsService,
+    protected routing: RoutingService
+  ) {}
+
+  order$: Observable<Order>;
+  returnableEntries: OrderEntry[] = [];
+  orderCode: string;
+
+  ngOnInit() {
+    this.order$ = this.orderDetailsService.getOrderDetails().pipe(
+      filter(order => !!order.entries),
```

Comment:
We are starting a new trend of using the Boolean constructor. So something like `Boolean(order.entries)`